spec/features/course_pages_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe 'CoursePages', type: :feature do
```

Comment:
I think the title should be as short as possible, because we need to rename file name also

OK I got what you mean, 'Courses management' makes sense